src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Slider, TextField } from '@material-ui/core';
+import { BasicInputWrapper } from '../BasicInputWrapper';
+import PropTypes from 'prop-types';
+
+const getValueText = (value) => {
+  return value.toString();
+};
+
+export const BasicSliderInput = (props) => {
+  const {
+    label,
+    tooltipText,
+    value,
+    handleSliderValueChange,
+    valueLabelDisplay,
+    step,
+    marks,
+    min,
+    max,
+    orientation,
+    disabled,
+    containerProps,
+    labelProps,
+    sliderProps,
+    color,
+    ...otherProps
+  } = props;
+  const getMarks = (marks) => {
+    return marks !== undefined
+      ? marks
+      : [
+          { value: min, label: min.toString() },
+          { value: max, label: max.toString() },
+        ];
+  };
+  return (
+    <BasicInputWrapper
+      label={label}
+      tooltipText={tooltipText}
+      containerProps={containerProps}
+      labelProps={labelProps}
+      {...otherProps}
+    >
+      {disabled ? (
+        <TextField value={value} disabled />
+      ) : (
+        <Slider
+          value={value}
+          style={sliderProps}
```

Comment:
`sliderProps` sounds too generic for the "style" prop of the Slider, maybe we could rename this prop to `sliderStyle` ?